benchmark_runner/common/template_operations/template_operations.py

```diff
@@ -181,7 +183,7 @@ def generate_yamls(self, scale: str = '', scale_nodes: list = [], redis: str = N
             if scale_nodes:
                 for bulk in bulks:
                     for num in bulk:
-                        p = Process(target=self.generate_yamls_internal, args=(str(num), scale_nodes[int(num/scale)], redis,))
+                        p = Process(target=self.generate_yamls_internal, args=(str(num), str(scale * len(scale_nodes)), scale_nodes[int(num/scale)], redis,))
```

Comment:
Again, just pass num and scale as integers, and let `generate_yamls_internal` convert them to strings.

Reply:
Not see it as problem